### ISSUE TITLE
Missing BWA file from GitHub Pull

### ISSUE THREAD

**Issue by foamy1881** [dsarov/SPANDx#17]
I pulled via github and install the SPANDx as required and upon running it have missing bwa file? 
yfong@yfong-VirtualBox:~/Documents/software/SPANDx$ ./SPANDx.sh -r NC_000962.3
Reference = NC_000962.3

The following parameters will be used

-------------------------------------

Organism = haploid
Output directory and directory containing sequence files = /home/yfong/Documents/software/SPANDx
SNP matrix will be created? = no
Genomes will be annotated? = no
Strain(s) to be processed = all
Sequence technology used = Illumina
Pairing of reads = PE
Variant genome specified for SnpEff = no
Window size for BedTools = 1000
Indels will be merged and corrected = no
-------------------------------------

Found FASTA file

FASTA file looks to contain no blank lines. Good.

ERROR: SPANDx requires BWA to function. Please make sure the correct path is specified in SPANDx.config

**Comment by ammaraziz** [comment]
Did you modify the `SPANDx.config` file to specify where `SPANDx` was installed? In the config file you should see this:

```
###
###       When installing SPANDx please modify the SPANDx_LOCATION variable to the installation path
###       SPANDx should run from any location but has been most extensively tested from /home/user/bin

### path to SPANDx installation  ####

SPANDx_LOCATION=/install/location/SPANDx_v3.0
```
Change where it says `SPANDx_LOCATION=/install/location/SPANDx_v3.0` to where appropriate. Eg for me it would be `SPANDx_LOCATION=/home/amar/bin/SPANDx/`

**Comment by dsarov** [comment]
This should also be fixed as of version 4.0 and with the conda distribution
